src/app/roles.tsx

```diff
@@ -27,22 +27,20 @@ const pathsOnlyShownToStaffAndAbove = ["/lists"] as const;
 const pathsOnlyShownToAdmin = ["/admin"] as const;
 
 const getShownPagesByRole = (role: UserRole | null): readonly string[] => {
-    switch (role) {
-        case "admin":
-            return [
-                ...pathsShownToAllAuthenticatedUsers,
-                ...pathsOnlyShownToStaffAndAbove,
-                ...pathsOnlyShownToAdmin,
-            ];
-        case "manager":
-            return [...pathsShownToAllAuthenticatedUsers, ...pathsOnlyShownToStaffAndAbove];
-        case "staff":
-            return [...pathsShownToAllAuthenticatedUsers, ...pathsOnlyShownToStaffAndAbove];
-        case "volunteer":
-            return pathsShownToAllAuthenticatedUsers;
-        case null:
-            return pathsNotRequiringLogin;
+    if (adminRoles.includes(role as AdminRolesType)) {
+        return [
+            ...pathsShownToAllAuthenticatedUsers,
+            ...pathsOnlyShownToStaffAndAbove,
```

Comment:
I feel like we should rename this constants to `pathsOnlyShownToOrganisationAndAbove` so it's consistent with the new role group naming system